.github/workflows/scripts/check-jira-release.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Queries Jira for the release date.
+#
+set -euo pipefail
+
+VERSION="$1"
```

Comment:
Also, let's agree on formatting and align all the scripts: [[ ]] or [ ], VAR or {VAR} etc. I don't have a preference.

Reply:
This may be difficult to agree on. `[]` `[[]]` and `(())` have different applications. So far, Malte suggested me that we should use `(())` for maths calc, then `[[]]` for all other things if we are in bash, and `[]` for all other cases.

`${VAR}` is the safest (can be used everywhere), but in many cases `$VAR` is enough and shorter.

Reply:
I would still insist on the `[[ -n "${VAR}" ]] || die "err msg"` here.